test/k8sT/manifests/guestbook_deployment.json

```diff
@@ -126,37 +126,50 @@
     }
 }
 {
-    "kind":"ReplicationController",
-    "apiVersion":"v1",
-    "metadata":{
-        "name":"guestbook",
-        "labels":{
-            "k8s-app.guestbook":"web"
-        }
+    "apiVersion": "apps/v1",
+    "kind": "Deployment",
+    "metadata": {
+        "name": "frontend"
     },
-    "spec":{
-        "replicas":1,
-        "selector":{
-            "k8s-app.guestbook":"web"
+    "spec": {
+        "selector": {
+            "matchLabels": {
+                "app": "guestbook",
```

Comment:
The selector in the policy in `guestbook-policy-web.yaml` should change as well to reflect the change in labels.